<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.12.1 on darwin)" generated="2024-11-13T18:05:29.130548" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="/Users/mr.heart_2711/.vscode/extensions/robocorp.robotframework-lsp-1.13.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-11-13T18:05:29.141282" elapsed="6.303110"/>
</kw>
<status status="PASS" start="2024-11-13T18:05:29.140965" elapsed="6.303785"/>
</test>
<status status="PASS" start="2024-11-13T18:05:29.131321" elapsed="6.314004"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-11-13T18:05:29.127675" level="WARN">Error in file '/Users/mr.heart_2711/.vscode/extensions/robocorp.robotframework-lsp-1.13.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
